site2/docs/functions-overview.md

```diff
@@ -31,19 +31,19 @@ The following is an example of a Pulsar Function written in Go.
 
 ```
 import (
-	"fmt"
-	"context"
+    "fmt"
```

Comment:
I think you shouldn't fix the tabs here. Golang uses `tabs` not `spaces`

Reply:
I have updated the PR by reverting the unintentional tab reformatting.